Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG version=3.8
+FROM python:${version} AS base  
+WORKDIR /app
+
+COPY . .
```

Comment:
Consider using a `.dockerignore` file to exclude unnecessary files and directories from being copied into the Docker image. This can help reduce the image size and improve build performance.